UNRELEASED_CHANGELOG.md

```diff
@@ -39,6 +39,11 @@
 ### ⬆️ Improved
 
 ### ✅ Added
+- Added new versions with API based on kotlin `StateFlow` for the following classes:
+  * `io.getstream.chat.android.offline.ChatDomain`
+  * `io.getstream.chat.android.offline.channel.ChannelController`
+  * `io.getstream.chat.android.offline.thread.ThreadController`
+  * `io.getstream.chat.android.offline.querychannels.QueryChannelsController`
```

Comment:
I'm not sure where and when we want to document these new APIs, but we should be linking to them here in the changelog.

Reply:
So you prefer not to mention this offline classes right now?

Reply:
I think when we ship these as public API & notify clients about it in the changelog, we should have all the relevant docs linked. Doesn't necessarily have to be added to the changelog on this same PR, I'm just generally asking about what the plan to document these APIs is.

Reply:
I created a new Jira task in epic. So we can postpone this discussion